demos/smartwatch/lv_demo_smartwatch.c

```diff
@@ -178,7 +178,11 @@ bool lv_demo_smartwatch_get_load_app_list(void)
 
 static void dialog_close_event_cb(lv_event_t * e)
 {
-    lv_obj_add_flag(dialog_window, LV_OBJ_FLAG_HIDDEN);
+    lv_event_code_t event_code = lv_event_get_code(e);
+    if(event_code == LV_EVENT_CLICKED) {
+        lv_obj_add_flag(dialog_window, LV_OBJ_FLAG_HIDDEN);
+    }
+
 }
```

Comment:
FYI, if you added this just to silence the unused parameter warning, you can also do `LV_UNUSED(e);`. It's ok to have unused parameters.

Reply:
got it, thanks. I wasn't sure what to do with the unused parameters